Source/Redemption/Private/SkullNPC.cpp

```diff
@@ -73,3 +80,20 @@ void ASkullNPC::Tick(float DeltaTime)
 
 }
 
+void ASkullNPC::PlayNextSound(int32 CurrentSoundIndex, int32 TotalSounds)
+{
+    if (CurrentSoundIndex < TotalSounds)
+    {
+        if (XyloSounds != nullptr)
+        {
+            float RandomPitch = FMath::RandRange(0.5f, 1.5f);
+            UGameplayStatics::PlaySoundAtLocation(this, XyloSounds, GetActorLocation(), FRotator::ZeroRotator, 1.0f, RandomPitch);
+        }
+
+        // Schedule the next sound playback after a delay
+        GetWorldTimerManager().SetTimer(SoundTimerHandle, [this, CurrentSoundIndex, TotalSounds]()
```

Comment:
how does this work?

Reply:
This part sets up the next sound playback after a 0.2-second delay using a timer.It calls the PlayNextSound function with the next sound index after that delay.

Reply:
Also I dont understand why theres a merge conflict i should have been the only one changing the bp. I don't know how to fix that

Reply:
the last time the BP_Talkingskull was changed was 10 days ago, I can only imagine that for some reason you you pulled it you did not receive it, or that the creation of the branch happened before the blueprint was merged to main

Reply:
I can help you with solving the conflict